scripts/o3de/o3de/github_utils.py

```diff
@@ -86,6 +87,86 @@ def clone_from_git(self, uri: ParseResult, download_path: pathlib.Path, force_ov
                 return 1
 
         return proc.returncode
+    
+    def upload_release(self, repo_uri: ParseResult, zip_path: pathlib.Path, archive_filename: str, upload_git_release_tag: str) -> int:
```

Comment:
The abstract method uses `git_release_tag` and this method overload should match it.
```suggestion
    def upload_release(self, repo_uri: ParseResult, zip_path: pathlib.Path, archive_filename: str, git_release_tag: str) -> int:
```